.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,75 @@
+
+name: Bug Report
+description: Create a bug-report to help us address errors and bugs.
+title: "🐛 [BUG] BUG TITLE HERE"
+labels: [🐛 bug 🐛, 🚦 status: awaiting triage 🚦, 🚧 status: blocked 🚧]
+assignees: 'j-dogcoder'
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A concise description of what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. In this environment...
+      2. With this config...
+      3. Run '...'
+      4. See error...
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Anything else?
+    description: |
+      Links? References? Anything that will give us more context about the issue you are encountering!
+      
+      Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  validations:
+    required: false
+
+---
+name: Bug Report
+about: Create a bug-report to help us address errors and bugs.
+title: '🐛 BUG: BUG TITLE HERE'
+labels: "🐛 bug 🐛, 🚦 status: awaiting triage 🚦, 🚧 status: blocked 🚧"
+assignees: 'j-dogcoder'
+
+---
+
+**Description**
+
+<!--A clear and concise description of what the bug is.-->
+
+**Error Code:**
+
+Code Terminal is logging:
+
+**Working Environment**
+
+Operating System (eg Linux):
+
+**Screenshots**
+
+Please add a screenshot if applicable
+
+[Optional] **Additional Context**
+
+Add any other context about the problem here.
```

Comment:
Uhh, I think this should be removed? This looks like a Markdown issue template.

Reply:
Ah, we should change it to be a markdown file then.